api/clients/v2/config.go

```diff
@@ -61,6 +61,24 @@ type RelayPayloadRetrieverConfig struct {
 	RelayTimeout time.Duration
 }
 
+// ValidatorPayloadRetrieverConfig contains an embedded PayloadClientConfig, plus all additional configuration values
+// needed by a ValidatorPayloadRetriever
+type ValidatorPayloadRetrieverConfig struct {
+	PayloadClientConfig
+
+	// The timeout duration for calls to retrieve blobs from a given quorum.
+	RetrievalTimeout time.Duration
+
+	// The address of the BlsOperatorStateRetriever contract
+	BlsOperatorStateRetrieverAddr string
+
+	// The address of the EigenDAServiceManager contract
+	EigenDAServiceManagerAddr string
+
+	// The number of simultaneous connections to use when fetching chunks during validator retrieval
+	ConnectionCount uint
```

Comment:
shouldnt this be a max count instead?

Reply:
[3f6f092c](https://github.com/Layr-Labs/eigenda/pull/1194/commits/3f6f092c87fa167c8b5d1462e2e6ffaa4c456785)